x-pack/solutions/security/packages/ecs_data_quality_dashboard/impl/data_quality_panel/actions/styles.tsx

```diff
@@ -5,9 +5,10 @@
  * 2.0.
  */
 
-import styled from 'styled-components';
+import { UseEuiTheme } from '@elastic/eui';
+import { CSSObject } from '@emotion/react';
 
-export const StyledLinkText = styled.span`
-  display: flex;
-  gap: ${({ theme }) => theme.eui.euiSizeS};
-`;
+export const linkTextCss = ({ euiTheme }: UseEuiTheme): CSSObject => ({
+  display: 'flex',
+  gap: euiTheme.size.s,
+});
```

Comment:
Instead of defining shared `useStyles` hooks unnecessarily, I decided to export small nuggets of shared css snippets like this. When passed directly into `css={}` prop as a function emotion passes in the theme for free so no need for import, also no need to wrap the properties in `css()` call. Imho, very handy and nifty.